src/Controller/Component/RequestHandlerComponent.php

```diff
@@ -47,14 +47,6 @@ class RequestHandlerComponent extends Component
     public $enabled = true;
 
     /**
-     * Holds the reference to Controller::$response
-     *
-     * @var \Cake\Network\Response
-     * @deprecated 3.4.0 Will be removed in 4.0.0
-     */
-    public $response;
```

Comment:
It is also declared on the base Component class. This definition was always redundant.